controllers/component_build_controller_pac.go

```diff
@@ -902,3 +1153,61 @@ func retrievePipelineSpec(bundleUri, pipelineName string) (*tektonapi.PipelineSp
 	pipelineSpec := pipelineSpecObj.PipelineSpec()
 	return &pipelineSpec, nil
 }
+
+func updateIncoming(repository *pacv1alpha1.Repository, incomingSecretName string, pacIncomingSecretKey string, targetBranch string) bool {
+	foundSecretName := false
+	foundTarget := false
+	multiple_incomings := false
+	all_targets := []string{}
+
+	if repository.Spec.Incomings != nil {
+		if len(*repository.Spec.Incomings) > 1 {
+			multiple_incomings = true
+		}
+
+		for idx, key := range *repository.Spec.Incomings {
+			// for multiple incomings gather all targets
```

Comment:
maybe this is a matter of taste, but to me such comments should be inside the block (in this case first line inside `if`). Otherwise I perceive them as comments to the if condition.

Reply:
it is comment about if / else

Reply:
There is another comment in if block that, imo, should be in else block